Composer/packages/extensions/visual-designer/src/components/menus/EdgeMenu.tsx

```diff
@@ -26,12 +31,44 @@ interface EdgeMenuProps {
   addCoachMarkRef?: (ref: { [key: string]: HTMLDivElement }) => void;
 }
 
+const buildCustomActionsSubmenu = (
+  label: string,
+  customizedActionGroups: any[][],
```

Comment:
Can this have a type?

Reply:
it's from the `oneOf` field of a schema file. I named it to `DefinitionSummary`